docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,215 @@
 # Getting started
 
-TODO
+In this tutorial, we will download and install Pebble, configure layers, run the Pebble daemon, and work with layers and services. After this tutorial, you will have a basic understanding of what Pebble is, how to install it, and how to use it to orchestrate services. It takes about 15 minutes to complete.
+
+
+After that, you can continue exploring more advanced features and use cases (links at the end).
+
+## Prerequisites
+
+- A Linux machine.
+
+## Download and install Pebble
+
+Find the latest tag on the [latest release page](https://github.com/canonical/pebble/releases/latest), then run the following commands to download, extract, and install the latest release (replace `v1.11.0` with the latest tag and `amd64` with your architecture):
+
+```bash
+wget https://github.com/canonical/pebble/releases/download/v1.11.0/pebble_v1.11.0_linux_amd64.tar.gz
+tar zxvf pebble_v1.11.0_linux_amd64.tar.gz
+sudo mv pebble /usr/local/bin/ # make sure it's in your $PATH
+```
+
+## Verify Pebble installation
+
+Once the installation is complete, verify that `pebble` has been installed correctly by running:
+
+```bash
+pebble
+```
+
+This should produce output similar to the following:
+
+```bash
```

Comment:
I strongly suggest using the terminal output directive for this. While this may just be 'formatting', it helps readers easily discern between commands that they expect to copy and paste, vs output that is meant to illustrate what happens.

Reply:
I suppose by "terminal output" you mean ```console```? I changed them all to preserve the colours.

If you mean using the myst terminal output, I'm pushing it back, for two reasons:

- Most of all, IIRC, as we discussed earlier with the team, we prefer to use markdown instead of myst.
- Markdown will render in most IDE/text editors out of the box, providing an easy live preview, but many myst directives can't, making editing and previewing at the same time difficult: the process has to be edit->build->preview->rinse and repeat.